content/en/chaos_engineering.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Chaos Engineering or CE is the discipline of experimenting on a [distributed system](https://glossary.cncf.io/distributed_systems/) in production to build confidence in the system's capability to withstand turbulent and unexpected conditions.
+
+## Problem it addresses
+[SREs](https://glossary.cncf.io/site_reliability_engineering/) and [DevOps](https://glossary.cncf.io/devops/) continuously look for new techniques to increase product resiliency and [reliability](https://glossary.cncf.io/reliability/). A system's ability to tolerate failures while ensuring adequate service quality is typically a software development requirement. But deep testing on a production system is not always possible due to the complexity of the architecture. High-frequency deployment of new features to the production environment can result in a high probability of downtime and a critical incident — with considerable consequences to the business.
```

Comment:
I don't really like to personify terms like SRE but especially DevOps. Can we change that to something like: "people working in the field of SRE and DevOps"?!.

Reply:
I agree, good one, thank you @iamNoah1